packages/console/src/monaco/lang/db.ts

```diff
@@ -42,7 +45,9 @@ const conf = {
   },
 };
 
-const language = {
+const language:
+  | monaco.languages.IMonarchLanguage
+  | monaco.Thenable<monaco.languages.IMonarchLanguage> = {
```

Comment:
You can just declare this as `monaco.languages.IMonarchLanguage`. `monaco.Thenable` is for async functions.
```suggestion
const language: monaco.languages.IMonarchLanguage = {
```